### ISSUE TITLE
invalid escape sequence '\s' in renpy/launcher/game/gui7/code.py on line 170

### ISSUE THREAD

**Issue by lordmatt** [renpy/renpy#5952]
The line in question from the class CodeGenerator on the update_defines method in the file `/usr/share/games/renpy/launcher/game/gui7/code.py` is `m = re.match('^(\s*)define (.*?) =', l)`. the error might have thrown during the deb install but this is a Renpy error.

I can't find my previous bug report about this. As I might have mentioned once or twice, being old and exhausted all the time is a significant drag factor on grepping my way past things like this.

I might be old but I stall know how to chase down an error message. Also I am old and my prime coding years are long past me - I haven't a clue what to do to fix this. It's looking for something but not getting what it expected. (Didn't we use to shout at noobs for not validating the data is in the form expected?)

According to the [re docs](https://docs.python.org/3/library/re.html), \s is a whitespace escape. But, if I remember my reg ex gotchas - backslash is a string escape character too.

I think you want:

`m = re.match('^(\\s*)define (.*?) =', l)`

It certainly runs without error with that change. Sadly, Renpy is still broken. Is this a regression or some upstream delay problem that the deb package has yet to update or a bug?

TL;DR: the .deb might be broken but renpy/launcher/game/gui7/code.py might be too. That or I am passed it and misunderstood what I saw. It's possible. I'm not as young as I used to be.

**Comment by renpytom** [comment]
Hi there. 

We don't produce a .deb package.
